backend/src/routes/edx-router.js

```diff
@@ -29,5 +29,7 @@ router.put('/exchange/:secureExchangeID/markAs', passport.authenticate('jwt', {s
 router.get('/exchange/:secureExchangeID', passport.authenticate('jwt', {session: false}, undefined), auth.isValidExchangeUserToken, extendSession, getExchange);
 router.post('/exchange/:secureExchangeID/comments', passport.authenticate('jwt', {session: false}, undefined), auth.isValidExchangeUserToken,extendSession, createSecureExchangeComment);
 router.post('/exchange', passport.authenticate('jwt', {session: false}, undefined), auth.isValidExchangeUserToken, extendSession, createExchange);
+router.post('/exchange/:secureExchangeID/documents', passport.authenticate('jwt', {session: false}, undefined), auth.isValidExchangeUserToken, extendSession, uploadDocumentToExchange);
+router.get('/exchange/:secureExchangeID/documents/:documentId', auth.isValidExchangeUserToken, getExchangeDocumentById());
```

Comment:
For the router.get documentId, I think we're missing passport.authenticate('jwt', {session: false}, undefined)

Reply:
I don't believe this works; I followed the same pattern as Student Profile there was a specific comment as to why this was required. I'll try to find it.

Reply:
@SoLetsDev - https://github.com/bcgov/EDUC-STUDENT-PROFILE/blob/7aca76d138cd7d3de6f737c9fecfeb66cdd73b8d/backend/src/routes/penRequest.js#L50

Reply:
@SoLetsDev Same one from staff side - I believe it's correct: https://github.com/bcgov/EDUC-STUDENT-ADMIN/blob/aed69b8c658d3e6a1f457b92ea71bccb9da3d2b2/backend/src/routes/penRequest.js#L52